content/docs/csidriver/installation/helm/powerstore.md

```diff
@@ -156,6 +158,7 @@ CRDs should be configured during replication prepare stage with repctl as descri
 | externalAccess | Defines additional entries for hostAccess of NFS volumes, single IP address and subnet are valid entries | No | " " |
 | kubeletConfigDir | Defines kubelet config path for cluster | Yes | "/var/lib/kubelet" |
 | imagePullPolicy | Policy to determine if the image should be pulled prior to starting the container. | Yes | "IfNotPresent" |
+| nfsAcls | Defines permissions - POSIX or NFSv4 ACLs, to be set on NFS target mount directory. | No | "0777" |
```

Comment:
The same feature has been named `isiVolumePathPermissions` in csi-powerscale : https://dell.github.io/csm-docs/docs/csidriver/features/powerscale/#helm-based-installation-1

Do you know if there is a plan to use the same name in csi-powerscale ? ?

Reply:
@coulof , I am not aware of any such plan.

Reply:
@coulof The parameter in csi-isilon stands specific to Isilon. Let me take it up with POs and get the parameter normalized